packages/ag-charts-community/src/chart/data/dataModel.ts

```diff
@@ -865,6 +1519,7 @@ export class DataModel<
 
         const columns: unknown[][] = [];
         const allColumnScopes: Set<ScopeId>[] = [];
+        const columnValueTypes: boolean[] = [];
```

Comment:
nit: I think the name and type don't intuitively match. `columnValueTypes: boolean[]`
Maybe rename to `columnNeedValueOf`? or something else...

Reply:
Done